lib/features/composer/presentation/widgets/email_address_input_builder.dart

```diff
@@ -227,6 +228,7 @@ class EmailAddressInputBuilder {
 
   Widget _buildSuggestionItem(StateSetter setState, BuildContext context, TagsEditorState<EmailAddress> tagEditorState, EmailAddress emailAddress) {
     return ListTile(
+      contentPadding: const EdgeInsets.symmetric(horizontal: 12),
```

Comment:
please do the same for autoComplete in `advancedSearch`

Reply:
we should limit the count of suggestions to 5, I think

Reply:
Advanced is using another widget. But its spacing is the same as `composer`.